pkg/cmd/roachtest/tests/logical_data_replication.go

```diff
@@ -442,32 +555,35 @@ func setupLDR(
 	t test.Test,
 	c cluster.Cluster,
 	setup multiClusterSetup,
-	kvWorkload replicateKV,
+	dbName string,
+	tableName string,
+	initCmd func(tenantName string, nodes option.NodeListOption) string,
```

Comment:
Might make more sense to have kvWorkload implement the streamingWorkload interface and continue to take the workload.